tests/zfs-tests/tests/functional/io/io_uring.ksh

```diff
@@ -44,6 +44,10 @@ if [[ $(linux_version) -lt $(linux_version "5.1") ]]; then
 	log_unsupported "Requires io_uring support"
```

Comment:
Looks like we can remove this check now, no?

Reply:
Good point.  Done.